WordPress/gutenberg
# ResizeTooltip: Use default.fontFamily on tooltip

## What?
Small PR to fix one of the issues reported as part of #48443. 

## Why?
The theme's body font family is incorrectly applied to the resize tooltip. The editor's UI should not be influenced by the theme's font family. 

## How?
Use `${ font( 'default.fontFamily' ) };` on resize tooltip styles.

## Testing Instructions

1. Switch to a theme using a noticeable body font (like Twenty Fifteen).
1. Open a Post or Page. 
2. Insert a Spacer block.
3. Resize it using the ResizableBox handle.
4. See the resize tooltip no longer using the body font from the theme.

## Screenshots or screencast 

Before: 
<img width="719" alt="CleanShot 2023-03-06 at 15 32 34" src="https://user-images.githubusercontent.com/1813435/223226885-058e3f15-f914-4f65-9a82-02f83287cb70.png">

After: 
<img width="724" alt="CleanShot 2023-03-06 at 15 30 39" src="https://user-images.githubusercontent.com/1813435/223226866-5759a928-8b7a-413b-95a1-7bb7f2bb6ef4.png">
